main/test/ca/uwaterloo/flix/language/feature/Test.Expression.ArrayLength.flix

```diff
@@ -0,0 +1,116 @@
+@test
+def testArrayLength01(): Int = let x = []; length[x]
+
+@test
+def testArrayLength02(): Int = let x = [1]; length[x]
+
+@test
+def testArrayLength03(): Int = let x = [1, 2, 3]; length[x]
+
+@test
+def testArrayLength04(): Int = let x = [1,2,3,4]; length[x]
```

Comment:
Any reason why you sometimes have spaces and sometimes not between elements? I think 1, 2, 3, 4 is easier to read.